ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1159,6 +1159,56 @@ def test_tensorflow_realdiv(
         y=x[1],
     )
 
+    
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    input_dtype, input_array, input_shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["int32", "int64"],
+            min_value=0,
+            max_value=10,
+            ret_shape=True
+        ))
+    input_dims = abs(len(input_shape))
```

Comment:
`abs` is  not needed. `len` is always non-negative.

Reply:
I met with an error which led me to conclude that this was a negative number. 
```Trying to create tensor with negative dimension -1: [-1]```

I forgot to remove this when I added your suggestion, and will add that commit now.